datapackage_pipelines_assembler/processors/load_private_resource.py

```diff
@@ -0,0 +1,30 @@
+import datapackage
+from datapackage_pipelines.wrapper import ingest
+from datapackage_pipelines.lib.load_resource import ResourceLoader
+from planner.utilities import s3_path
+
+
+class PrivateResourceLoader(ResourceLoader):
+
+    def __init__(self):
+        super(ResourceLoader, self).__init__()
+        self.parameters, self.dp, self.res_iter = ingest()
+
+    def process_datapackage(self, dp_):
+        cached_paths = {}
+        for i, res in enumerate(dp_.resources):
```

Comment:
can you please explain the issue here?
Why not just set the `path` in each resource?

Reply:
@akariv I tried to set both `path` and `url` to be pre-signed URLs but when loading resouce it is still using URL that is assigned to `res.source`.

Reply:
+ even it works, if we leave path to  be signed URL we will end up with datapackage.json with `path:signed_url` -> this will lead eg. zip files with paths like `https:/amazonaws/pkgstore/datahub.io/...` So I found this way the best to sign URL for resources and keep original paths